compareECLFiles.cmake

```diff
@@ -263,20 +263,22 @@ add_test_compareECLFiles(CASENAME polymer_oilwater
                          FILENAME 2D_OILWATER_POLYMER
                          SIMULATOR flow
                          ABS_TOL ${abs_tol}
-                         REL_TOL ${rel_tol})
+                         REL_TOL ${rel_tol}
+                         TEST_ARGS --tolerance-mb=1.e-7 )
 
 add_test_compareECLFiles(CASENAME polymer_injectivity
                          FILENAME 2D_POLYMER_INJECTIVITY
                          SIMULATOR flow
                          ABS_TOL ${abs_tol}
-                         REL_TOL ${rel_tol})
+                         REL_TOL ${rel_tol}
+                         TEST_ARGS --tolerance-mb=1.e-7 --max-strict-iter=20 --tolerance-wells=1.e-6)
```

Comment:
Do we know that all three are necessary? I.e., if you remove on of them, will the test still pass on your machine?

Reply:
`--tolerance-mb=1.e-7 --max-strict-iter=20 --tolerance-wells=1.e-6` were added in the order as it shows. I can also test it by removing one of the first two.  Only the third one will not work.